db/models.py

```diff
@@ -1 +1,42 @@
 from django.db import models
+
+
+class Race(models.Model):
+    RACE_CHOICES = [
+        ("elf", "Elf"),
+        ("dwarf", "Dwarf"),
+        ("human", "Human"),
+        ("ork", "Ork"),
+    ]
+
+    name = models.CharField(max_length=255, unique=True, choices=RACE_CHOICES)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
```

Comment:
The `race` ForeignKey in the `Skill` model should have a `related_name` attribute to facilitate reverse lookups. This is important for accessing related `Skill` instances from a `Race` instance.